components/Buttons/tests/unit/ButtonsThemingTest.swift

```diff
@@ -84,4 +84,29 @@ class ButtonsThemingTest: XCTestCase {
     XCTAssertEqual(buttonShape.bottomRightCorner, shapeScheme.smallComponentShape.bottomRightCorner)
     XCTAssertEqual(buttonShape.bottomLeftCorner, shapeScheme.smallComponentShape.bottomLeftCorner)
   }
+  
+  func testFloatingButtonSecondaryTheme() {
```

Comment:
This looks like a test with color + typography but without shape.  Please add the other 6 test cases if that's how we're handling things:

1. No custom themes
2. Custom color only
3. Custom typography only
4. Custom shape only
5. Custom color, typography (this test)
6. Custom color, shape
7. Custom typography, shape


However, I would strongly recommend testing only defaults (no customization), each scheme in isolation, and possibly all 3 schemes.

1. No custom schemes
2. Custom color only
3. Custom typography only
4. Custom shape only
5. [Optional] All 3 custom schemes

Reply:
The shape testing was included, which followed the examples under shape testing. Will add the custom ones.

Reply:
I addressed the custom color, typography cases. The custom shape case was not added, because the shape theme for FAB doesn't use things from shapeScheme, which mean there is no way to customize it. I can add a case to verify no values from shapeScheme is used if you would prefer. Originally there were no testing for that when the FAB themer was implemented.

Reply:
I'm not sure how we could effectively test that the shape scheme "wasn't" used. Ideally, the shape scheme would apply the `cornerRadius` value for the buttons if that was the minimally-necessary condition for achieving the state. Although currently it might apply a masking layer to the component, testing for that would be very white-box testing of *another component*. I think it's best to leave it out for now.  Thanks for explaining!

Reply:
Thanks for creating another PR to make it clear.